wdl-engine/src/inputs.rs

```diff
@@ -57,7 +57,7 @@ fn join_paths<'a>(
             v.visit_paths_mut(false, &mut |_, v| {
                 v.expand_path()?;
                 v.join_path_to(path);
-                v.ensure_path_exists(false)
```

Comment:
Previously this was ensuring inputs to the first task or workflow; now task and workflow evaluation properly handle checking for existence (non-existence is also allowed for `File?` and `Directory?` inputs).

Reply:
This comment is no longer applicable; added this check back as required `File` and `Directory` inputs is a 1.2+ behavior.